src/constants/tokens.ts

```diff
@@ -134,39 +134,45 @@ export const LUSD_TOKEN = new Token({
   purchaseUrl: "",
 });
 
-export const OHM_WETH_LP_TOKEN = new LPToken({
+export const OHM_WETH_LP_TOKEN = new Token({
   decimals: 18,
   name: "OHM-WETH LP",
   icons: ["OHM", "wETH"],
   factory: PairContract__factory,
-  tokens: [OHM_TOKEN, WETH_TOKEN],
   addresses: OHM_WETH_LP_ADDRESSES,
   purchaseUrl:
     "https://app.sushi.com/add/0x64aa3364F17a4D01c6f1751Fd97C2BD3D7e7f1D5/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
```

Comment:
this could be dynamic url with ohm token / weth token as the path

Reply:
Done